README.md

```diff
@@ -409,6 +418,15 @@ assert.equal(
 );
 ```
 
+In addition, you can transform keys entirely without writing a custom encoder using `mapKeys`:
```

Comment:
how does this interact with a custom encoder? are the options mutually exclusive, or is there a specific order they run in? (same question re a custom decoder in parse)